### DISCUSSION TITLE
Security - Restricting use of Wildcard for APIGroup in clusterRole

### DISCUSSION THREAD

**Discussion by emirot** [argoproj/argo-cd#10403] [General]
Hey there,

In couple of manifest (https://github.com/argoproj/argo-cd/blob/master/manifests/install.yaml#L9505, https://github.com/argoproj/argo-cd/blob/master/manifests/install.yaml#L9525) we use the "*". Which is not the best on a security standpoint
Could that be improved? 
Would it be possible to just use "" that would mean only access to k8s API core? Or it will affect argocd functionalities.

**Comment by crenshaw-dev** [comment]
The Application controller needs access to anything you plan to manage with Argo CD. I think you could theoretically narrow down the RBAC to allow access only to the resources you plan to manage with Argo CD. But I haven't tested that personally.
